File: generator/output/entityModel/AdminModel.java
package com.licenseair.backend.domainModel;

import java.util.List;
import lombok.Data;

import java.lang.String;
import java.lang.Boolean;
import java.sql.Timestamp;

/**
* Created by licenseair.com
* Admin
*/
@Data()
public class AdminModel {
  public Long id = null;

  /**
  * 是否删除
  */
  public Boolean deleted = null;

  /**
  * 
  */
  public String email = null;

  /**
  * 
  */
  public String mobile = null;

  /**
  * 
  */
  public String password = null;

  /**
  * 是否激活
  */
  public Boolean active = null;

  /**
  * 网站显示
  */
  public String username = null;

  public Timestamp created_at = null;

  public Timestamp updated_at = null;

}

File: generator/output/ts-model/admin.model.ts
class AdminModel {
  public id: number = 0;

  // deleted 是否删除
  public deleted: boolean = false;

  // email 
  public email: string;

  // mobile 
  public mobile: string;

  // password 
  public password: string;

  // active 是否激活
  public active: boolean = false;

  // username 网站显示
  public username: string;

  public created_at: string;

  public updated_at: string;
}

const AdminModelFields = {
  id: "id",
  deleted: "deleted",
  email: "email",
  mobile: "mobile",
  password: "password",
  active: "active",
  username: "username",
  created_at: "created_at",
  updated_at: "updated_at"
};

export {AdminModel, AdminModelFields};
